core/controllers/admin.py

```diff
@@ -442,6 +446,39 @@ def _load_dummy_new_structures_data(self):
         else:
             raise Exception('Cannot load new structures data in production.')
 
+    def _generate_dummy_skill_and_questions(self):
+        """Generate and loads the database with a skill and 15 questions
+        linked to the skill.
+
+        Raises:
+            Exception: Cannot load new structures data in production mode.
+            Exception: User does not have enough rights to generate data.
+        """
+        if constants.DEV_MODE:
+            if self.user.role != feconf.ROLE_ID_ADMIN:
+                raise Exception(
+                    'User does not have enough rights to generate data.')
+            skill_id = skill_services.get_new_skill_id()
+            skill_name = 'Dummy Skill %s' % python_utils.UNICODE(
+                random.getrandbits(32))
+            skill = self._create_dummy_skill(
+                skill_id, skill_name, '<p>Dummy Explanation 1</p>')
+            skill_services.save_new_skill(self.user_id, skill)
```

Comment:
@rohitkatlaa, I just wanted to make sure that these skills also creates an opportunity for question suggestion.

 You can test this Following these steps:
1. Enable COMMUNITY_DASBOARD_PAGE_ENABLED flag in feconf.
2. Create dummy skills through the admin page.
2. Go to /community_dashboard page and check the "Questions" tab.

Reply:
@DubeySandeep I tried what you had asked me but I don't see any questions being displayed in the questions tab.

Reply:
Can you please check how skillOpprtunity is created and make sure that the dummy skill also creates skillOpportunities?

/cc @sagangwee

Reply:
@DubeySandeep I am not sure how to create skillOpportunities manually because I tried creating skills with questions and nothing is displayed in the community dashboard, is there some option that I have to click to make it appear there?

Reply:
I think only the skills which are linked to topics as treated as an opportunity. If you are creating a skill that is getting linked to a topic then it should generate an opportunity.

@sagangwee, Can you please confirm this?

Reply:
@DubeySandeep I created a skill and assigned to a topic but nothing was displayed in the community dashboard but I checked the skillOppurtunityModel in the datastore and found that the opportunities are being created even they are not linked to a topic. I also tested the button and it creates the opportunity in the datastore but nothing is displayed in the community dashboard.

Reply:
SkillOpportunities should be created whenever a skill is saved [here](https://github.com/oppia/oppia/blob/develop/core/domain/skill_services.py#L444), regardless if the skill is assigned to a topic.